source/fundamentals/connection.txt

```diff
@@ -0,0 +1,510 @@
+================
+Connection Guide
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. toctree::
+
+   /fundamentals/connection/tls
+
+This guide shows you how to connect to a MongoDB instance or replica set
+deployment using the driver. For information about authenticating
+with a MongoDB instance, see :ref:`Authentication <authentication>`.
+
+.. _mongoclient:
+
+MongoClient
+-----------
+
+Use the ``MongoClients.create()`` method to construct a
+``MongoClient``. As each ``MongoClient`` represents a threadsafe *pool* of
```

Comment:
Nitpick: if you provide emphasis on a term, I would expect an explanation of the term to follow. I would either:
1) omit the emphasis or 
2) explain the term